specification/servicelinker/ServiceLinker.Management/models.tsp

```diff
@@ -0,0 +1,1461 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.ServiceLinker;
+
+/**
+ * The name of action for you dryrun job.
+ */
+union DryrunActionName {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  createOrUpdate: "createOrUpdate",
+}
+
+/**
+ * The type of dryrun result.
+ */
+union DryrunPrerequisiteResultType {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  basicError: "basicError",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  permissionsMissing: "permissionsMissing",
+}
+
+/**
+ * The operation type
+ */
+union DryrunPreviewOperationType {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  configConnection: "configConnection",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  configNetwork: "configNetwork",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  configAuth: "configAuth",
+}
+
+/**
+ * The target service type.
+ */
+union TargetServiceType {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
```

Comment:
For enum values that dont have documentation, can our tooling just output the name of the value as docs?

Reply:
A small fix like that in our tooling would greatly improve this output and reduce the number of suppressions in the specs